package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "leaderboard",
+  "version": "1.0.0",
+  "description": "![](https://img.shields.io/badge/Microverse-blueviolet)",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
+  },
```

Comment:
- [ ] kindly add 'start' script so that you are able to automatically open application on browser by typing `npm start` command: 

```
 "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1",
    "build": "webpack",
    "start": "webpack serve --open"
  },
```

Reply:
Thank you, I would be sure to add it